live-examples/js-examples/object/object-entries.html

```diff
@@ -8,7 +8,12 @@
 // expected output: Array ["2", "c"]
 
 const object3 = { 100: 'a', 2: 'b', 7: 'c' };
-console.log(Object.entries(object3)[0]);
+//Object.entries() may or may not  return the objects in a particular order, Sort is needed
+const result = Object.entries(object3).sort((a,b) => a - b);
+console.log(Object.entries(result));
+// expected output: Array [Array ["2", "b"], Array ["7", "c"], Array ["100", "a"]]
+
+console.log(Object.entries(result)[0]);
 // expected output: Array ["2", "b"]
```

Comment:
With this addition, the example code is getting a bit long. To avoid that, I would suggest the following changes to the above:

```suggestion
const result = Object.entries(object2).sort((a, b) => a - b);
console.log(Object.entries(result)[1]);
// expected output: Array ["1", Array ["1", "b"]]
```